README.md

```diff
@@ -21,7 +21,7 @@
 
```

Comment:
- You have done great so far :+1: :+1: But there are some requirements you still need to meet before moving to the next milestone. :+1: 
- [ ] As the aim of this milestone is to follow the best practices for KISS, DRY code, I'll suggest you move the choice selection to your `main.rb`. `main.rb` is only to start their code and ask the user for the initial choice of options (or to quit).

- - Screenshot:
![Screenshot from 2023-08-14 23-29-34](https://github.com/Gardimy/OOP-school-library/assets/107632780/38ee5593-7d75-4838-8bfe-9153eee440ee)